aspnetcore/mvc/views/razor.md

```diff
@@ -54,19 +54,23 @@ Implicit Razor expressions start with `@` followed by C# code:
 <p>@DateTime.IsLeapYear(2016)</p>
 ```
 
-With the exception of the C# `await` keyword, implicit expressions must not contain spaces. You can intermingle spaces if the C# statement has a clear ending:
+With the exception of the C# `await` keyword, implicit expressions must not contain spaces. If the C# statement has a clear ending, you can intermingle spaces:
 
 ```cshtml
 <p>@await DoSomething("hello", "world")</p>
 ```
 
-Implicit expressions can't contain C# generics, as the characters inside the brackets (`<>`) are interpreted as an HTML tag:
+Implicit expressions **cannot** contain C# generics, as the characters inside the brackets (`<>`) are interpreted as an HTML tag. The following code is **not** valid:
 
 ```cshtml
 <p>@GenericMethod<int>()</p>
 ```
 
-The code generates a compiler error like `The "int" element was not closed.  All elements must be either self-closing or have a matching end tag` or `Cannot convert method group 'GenericMethod' to non-delegate type 'object'. Did you intend to invoke the method?` Instead, generic method calls must be wrapped in an [explicit Razor expression](#explicit-razor-expressions) or a [Razor code block](#razor-code-blocks). Note that this restriction doesn't apply to *.vbhtml* Razor files because Visual Basic syntax places parentheses around generic type parameters instead of brackets.
+The preceding code generates a compiler error similar to the following:
+
+* The "int" element was not closed.  All elements must be either self-closing or have a matching end tag or Cannot convert method group 'GenericMethod' to non-delegate type 'object'. Did you intend to invoke the method?
```

Comment:
Block quote? (`>`) ... Will the single asterisk here break?

Reply:
Also "Cannot convert" is the start of a second compiler error.  That should maybe be in a second blockquote